### ISSUE TITLE
Bug with the naming. (Text input broken)

### ISSUE THREAD

**Issue by krisz2000** [mk-dev-team/mk-world-of-imagination#3]
**Describe the bug**
When you want to name your world or change it's seed, the cursor gets placed at the forst character instead of the last one, and when your try to delete with the Delete key it only replaces the letters or numbers with an empty character.

**To Reproduce**
1. Go to new world in the title screen
2. Click into any box with text in it.
3. Press Delete.

**Expected behavior**
The deleted character only gets replaced.

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: 10

**Additional context**
It would be great if you could also speed up the time waited between character deletes when you hold down backspace.

**Comment by sleepy-monax** [comment]
Yeah, the text field is kinda broken he need a rewrite for the next update. Thanks for your bug repport
